app/src/main/java/com/infomaniak/drive/ui/menu/settings/SyncSettingsActivity.kt

```diff
@@ -248,14 +246,13 @@ class SyncSettingsActivity : BaseActivity() {
     private fun observeSelectedDrive() = with(binding) {
         selectDriveViewModel.selectedDrive.distinctUntilChanged().observe(this@SyncSettingsActivity) {
             it?.let {
-                driveIcon.imageTintList = ColorStateList.valueOf(Color.parseColor(it.preferences.color))
-                driveName.text = it.name
+                selectDrive.setColorFolder(Color.parseColor(it.preferences.color))
+                selectDrive.setTitle(it.name)
                 selectDivider.isVisible = true
                 selectPath.isVisible = true
             } ?: run {
-                driveIcon.imageTintList =
-                    ColorStateList.valueOf(ContextCompat.getColor(this@SyncSettingsActivity, R.color.iconColor))
-                driveName.setText(R.string.selectDriveTitle)
+                selectDrive.setColorFolder(ContextCompat.getColor(this@SyncSettingsActivity, R.color.iconColor))
```

Comment:
Maybe the naming of `setColorFolder` is not appropriate and should be more generic and not mention "folder" directly because here it's not used to color a folder